app/controllers/users_controller.rb

```diff
@@ -1,3 +1,27 @@
 class UsersController < ApplicationController
-  def new; end
+  def show
+    @user = User.find_by id: params[:id]
+    # redirect_to :root, flash: {warning: t(".user_not_found")} if @user.nil?
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
```

Comment:
```suggestion
    @user = User.new user_params
```
hạn chế ngoặc, chỗ nào bỏ dc thì bỏ em nhé

Reply:
E đã fix r ạ